src/dll/example/Makefile

```diff
@@ -0,0 +1,63 @@
+# ##########################################################################
```

Comment:
The whole `dll` folder should probably not be there

Reply:
👍